src/sentry/static/sentry/app/views/settings/organizationFiltersAndSampling/rules/rule/projects.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import PlatformList from 'app/components/platformList';
+import {PlatformKey} from 'app/data/platformCategories';
+import {t} from 'app/locale';
+import {Project} from 'app/types';
+import withProjects from 'app/utils/withProjects';
+
+type Props = {
+  projectIds: Array<number>;
+  projects: Array<Project>;
+};
+
+function Projects({projectIds, projects}: Props) {
+  if (!projectIds.length) {
+    return <React.Fragment>{t('All')}</React.Fragment>;
+  }
+
+  const projectPlatforms = projects
+    .filter(project => projectIds.includes(Number(project.id)))
+    .map(({platform}) => platform ?? 'other') as Array<PlatformKey>;
+
+  return <PlatformList platforms={projectPlatforms} />;
```

Comment:
Are we sure that withProjects will always have all projects?

You might need to check out:
https://github.com/getsentry/sentry/blob/b8bf85948d4c4832c60ab31e9f1a5a3c39fd53cf/src/sentry/static/sentry/app/utils/projects.tsx

Reply:
nice. Thanks!!